jbmc/src/java_bytecode/java_bytecode_parse_tree.h

```diff
@@ -98,6 +98,8 @@ struct java_bytecode_parse_treet
       return instructions.back();
     }
 
+    std::vector<annotationst> parameter_annotations;
```

Comment:
Please document this field (for a change in this file :)

Say, e.g., that the positions in the vector correspond to the positional formal arguments of the method represented by containing class.